packages/core/src/sync-realtime/index.ts

```diff
@@ -356,46 +413,20 @@ export const createRealtimeSync = (
       chainId: args.network.chainId,
     });
 
-    // Get all addresses from `logs` that are from a factory
-    const childAddresses = new Set<Address>();
-    for (const filter of factories) {
-      if (logs.length > 0) {
-        const _childAddresses = await args.syncStore.filterChildAddresses({
-          filter,
-          addresses: logs.map(({ address }) => address),
-        });
-        for (const address of _childAddresses) childAddresses.add(address);
-      }
-    }
-
     // Remove logs that don't match a filter
     logs = logs.filter((log) =>
-      logFilters.some((filter) => {
-        const isMatched = isLogFilterMatched({ filter, block, log });
-
-        if (isAddressFactory(filter.address)) {
-          return isMatched && childAddresses.has(log.address);
-        }
-
-        return isMatched;
-      }),
+      logFilters.some((filter) => isLogFilterMatched({ filter, block, log })),
```

Comment:
Is it worthwhile to filter here if we're going to filter again later on? The factory filtering _must_ happen sequentially (can't happen in `extract` which runs in parallel), so it might make sense to do all filtering in handleBlock? Don't feel strongly either way.